lucille-core/src/main/java/com/kmwllc/lucille/core/Runner.java

```diff
@@ -20,72 +20,77 @@
 import java.util.concurrent.TimeUnit;
```

Comment:
can we please avoid reformatting this whole file
this is too much change for this PR and I cannot review every single diff here

Reply:
:) fixed